### PR TITLE
add windows CI

### PR DESCRIPTION
**Pull request by zimbatm**
adds some form of testing for the Windows target.

Let's wait until we have more CI workers available before merging this PR.

---
This change is [Reviewable](https://reviewable.io/reviews/digital-asset/daml/207)

### CODE REVIEW

**Comment by dasormeter** [comment]
@zimbatm : this needs a rebase

File: ci/build-windows.yml [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
-    fetchDepth: 1 # speed up checkout from 30sec to 7sec
-  - powershell: '.\dev-env\windows\dadew-installer.ps1'
-    displayName: 'Install dev-env'
-    #- script: ./ci/configure-bazel.ps1
-    #  displayName: 'Configure Bazel'
-  - powershell: '.\build.ps1'
-    displayName: 'Bazel Build'
+    fetchDepth: 2 # speed up checkout from 30sec to 7sec
+
+  - powershell: |
```
Comment (majcherm-da) [inline ci/build-windows.yml]: I made `install/sync/enable` part of the `build.ps1`: https://github.com/digital-asset/daml/blob/master/build.ps1#L4

File: ci/build-windows.yml [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+
+  - bash: ci/configure-bazel.sh
+    displayName: 'Configure Bazel'
+    env:
+      IS_FORK: variables['System.PullRequest.IsFork']
+      # to upload to the bazel cache
+      GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
+
+  - powershell: |
+      .\dev-env\windows\bin\dadew.ps1 enable
```
Comment (majcherm-da) [inline ci/build-windows.yml]: as above

File: dev-env/windows/libexec/core.ps1 [hunk 3]
```diff
@@ -28,6 +28,8 @@ function da_reset_path {
 
 function da_enable_win_dev_env {
     da_enable_scoop
+    # also add dadew to the PATH
+    $env:PATH = "$PSScriptRoot\..\bin;$env:PATH"
```
Comment (majcherm-da) [inline dev-env/windows/libexec/core.ps1]: I combined it into: https://github.com/digital-asset/daml/blob/master/dev-env/windows/libexec/core.ps1#L20

**Comment by majcherm-da** [comment]
Finished as part of: https://github.com/digital-asset/daml/pull/364

**Pull request closed without merging**